crates/bevy_utils/src/default.rs

```diff
@@ -0,0 +1,25 @@
+/// Used to make initializing structs with defaults easier:
```

Comment:
```suggestion
/// An ergonomic abbreviation for [`Default::default()`] to make initializing structs easier
```
IMO this is a clearer, and gets to the heart of the magic quickly for experienced Rust users.